src/maps/maps.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { Client } from '@googlemaps/google-maps-services-js';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class MapsService {
+  client: Client;
```

Comment:
you can use JS notation of private variable # -> https://www.javascripttutorial.net/javascript-private-methods/